XUtils.c

```diff
@@ -303,3 +304,8 @@ ssize_t xReadfileat(openat_arg_t dirfd, const char* pathname, void* buffer, size
 
    return readfd_internal(fd, buffer, count);
 }
+
+int xClock_gettime (clockid_t __clock_id, struct timespec *__tp) {
+   return clock_gettime (__clock_id, __tp);
```

Comment:
What's the added value of having this wrapper? Also `__` prefixed identifiers are reserved for the C implementation and/or denote internal stuff. You could just call it `clock_id` and `tp` directly here.